policy-orchestration/README.md

```diff
@@ -0,0 +1,296 @@
+---
+type: general
+stakeholders:
+    - sig-architecture
+    - team-shield
+issue: "tbd"
+---
+
+# Policy Orchestration System
+
+> [!NOTE]
+> :bulb: This RFC describes a subsystem of the security platform which is an intended long-term approach to improving policy management.
+
+## :beginner: Current state / context
+
+Giant Swarm manages several third-party applications which provide security features to our customers.
+
+This management includes a set of default policies for each tool, which we recommend, deploy, and manage based on the perceived value for most / all customers.
```

Comment:
Maybe an example, reference to a existing policy, can help to understand what do you mean